front/app/api/authentication/singleSignOn.ts

```diff
@@ -28,6 +28,7 @@ export interface SSOParams {
   sso_verification_id?: string;
   sso_verification_type?: string;
   error_code?: SignUpInError;
+  verification_success?: string;
 }
```

Comment:
I don't have support to make BE changes right now, so I think the solution we need right now is to add verification_success here + add some handling for this case in the current SSO code.

Reply:
I think this is the best possible solution now, without BE capacity. Maybe we should add another comment to remind ourselves to clean this up some day, or log it somewhere- I have a lot of small improvements to the auth flow that I would like to make some day